packages/language/src/preprocessor/compiler-options/translator.ts

```diff
@@ -1353,21 +1353,26 @@ translator.flag("goff", ["GOFF"], ["NOGOFF"]);
 translator.rule(["INCAFTER"], (option, options) => {
   ensureArguments(option, 1, 1);
   const value = option.values[0];
-  ensureType(value, "option");
-  if (value.name.toUpperCase() !== "PROCESS") {
-    throw new TranslationError(
-      value.token,
-      `Expected "PROCESS", but received '${value.name}'.`,
-      1,
-    );
+  if (value.kind === SyntaxKind.CompilerOption) {
+    ensureType(value, "option");
```

Comment:
Don't we already know that the `kind` is an `option`? We shouldn't need to verify it anymore.